gulp/tasks/examples.js

```diff
@@ -41,7 +41,7 @@ webpackConfig = {
 		]
 	},
 	plugins: [
-
+		"transform-react-jsx"
```

Comment:
not 100% sure we actually need this, but I guess React deprecated it's own one circa v13 -- they recommend using this now (see https://facebook.github.io/react/blog/2015/06/12/deprecating-jstransform-and-react-tools.html#jsxtransformer)